UPGRADE.md

```diff
@@ -0,0 +1,71 @@
+### Kubevirt HCO upgrade procedure
+
+##### Summary
+
+In order to tell the OLM to start the upgrade process, one needs to point the relevant operator subscription to the updated CSV (ClusterServiceVersion).
+
+It may be as simple as updating the subscription's **channel**, inside the currently used **catalog**. Or, in other cases, it may require updating the **catalogsource** first, to point to the updated image, one that will contain a new catalog, with a CSV for this new channel.
```

Comment:
```suggestion
It may be as simple as updating the subscription's **channel**, inside the currently used **catalog**. In other cases, it may require updating the **catalogsource** first, to point to an updated image, one that will contain a new catalog, with a CSV for this new channel.
```